OpenIPSL/Examples/SMIBpartial.mo

```diff
@@ -47,7 +47,7 @@ partial model SMIBpartial "SMIB system with one load"
     annotation (Placement(transformation(extent={{-40,-10},{-20,10}})));
   inner OpenIPSL.Electrical.SystemBase SysData(S_b = 100e6, fn = 50)
     annotation (Placement(transformation(extent={{-100,80},{-40,100}})));
-  OpenIPSL.Electrical.Buses.Bus LOAD
+  OpenIPSL.Electrical.Buses.Bus LOAD(v_0=0.9919935, angle_0=-0.5762684)
```

Comment:
I'm not sure that setting some arbitrary flow values to a base class that should be used in different examples is a robust solution. These values might cause problems in other examples since it is used as a based. So if an example has a problem with the initialisation that that particular example should maybe be fixed individually.

Reply:
@dietmarw this Bus called LOAD is connected to a load with that voltage and angle initial values. The partial SMIB has the same power flow (and therefore node solution) for all cases where it is used. The flow values that I put in the bus are, therefore, not arbitrary. They are exactly the values for the solution and for the correct initialization of every SMIB. The conflict during initilization was just because the Bus and the Load were using different values and now they are using the same.